src/celestia/kde/kdepreferencesdialog.cpp

```diff
@@ -242,7 +242,7 @@ KdePreferencesDialog::KdePreferencesDialog(QWidget* parent, CelestiaCore* core)
     actionColl->action("showSpacecraftOrbits")->connect(showSpacecraftOrbitsCheck, SIGNAL(clicked()), SLOT(activate()));
     showSpacecraftOrbitsCheck->setChecked(orbitMask & Body::Spacecraft);
     labelGroupLayout->addWidget(showSpacecraftOrbitsCheck, 12, 0, Qt::AlignHCenter);
-    QCheckBox* showSpacecraftLabelsCheck = new QCheckBox(i18n("Spacecrafts"), labelGroup);
+    QCheckBox* showSpacecraftLabelsCheck = new QCheckBox(i18n("Spacecraft"), labelGroup);
```

Comment:
kde frontend is not maintained, so I think you can just leave it here

Reply:
Since I already changed it I see no need to revert just because the frontend isn't maintained.